docs/DeveloperGuide.md

```diff
@@ -171,6 +171,50 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### The onboarding guide
```

Comment:
minor nit, capitalise the text here since it is the header
```suggestion
### The Onboarding Guide
```

Reply:
Fixed